bioimageio/spec/io_.py

```diff
@@ -170,7 +172,35 @@ def load_raw_resource_description(
                 f"(always updating to latest patch version)."
             )
 
-    sub_spec = _get_spec_submodule(type_, data_version)
+    try:
+        sub_spec = _get_spec_submodule(type_, data_version)
+    except ValueError as e:
+        try:
+            sub_spec = _get_spec_submodule(type_, data_version=LATEST)
+        except ValueError:
+            raise e  # raise original error with desired data_version
+
+        try:
+            odv = Version(original_data_version)
+        except Exception as e:
+            raise ValueError(f"Invalid format version {original_data_version}.") from e
+
+        if Version(sub_spec.format_version) < odv:
+            warnings.warn(
+                f"Loading future {type_} format version {original_data_version} as (latest known) "
+                f"{sub_spec.format_version}."
+            )
+            data["format_version"] = sub_spec.format_version
+            if "config" not in data:
```

Comment:
cool, could you comment why you add `"config"` and `"config/biomimageio"` key? just for context - I guess they were required starting from some version?

Reply:
I added some comments to clarify... 
I need to optional 'config/bioimageio' dict only because I wanted to save the original format version for reference

Reply:
we could also not do that and remove this part. I thought it might help debugging, etc...

Reply:
ah, no, don't remove it. My first instinct was that you added it to conform some kind of implicit schema, but if it's for debugging (and if you state that in a comment) I think it's also comprehensible for someone less familiar with the code.

Reply:
one could even do some overkill explicit naming with `debug` something in the key... or not :)

Reply:
debug might not be the only use case... so I left it as what it is `original_format_version` and put it in the `bioimageio` part of the config to avoid key clashes across different software (as is the recommendation)